docs/getting-started/codacy-quickstart.md

```diff
@@ -49,23 +49,9 @@ Click the link **Go to repository** to see the [code quality overview of your re
 
 ![Repository dashboard](../repositories/images/repository-dashboard.png)
 
-## 4. Tweak your repository settings {: id="tweak-settings"}
-
- Check the static analysis results on the **Issues** page to [explore the initial analysis results](../repositories/issues.md).
-
- The first analysis is based on Codacy **default configurations**, so you should tweak and adapt the following configurations to match your preferences and specific scenario:
-
--   [Ignore files](../repositories-configure/ignoring-files.md) that you want to exclude from the Codacy analysis
--   [Configure the code patterns](../repositories-configure/configuring-code-patterns.md) that Codacy uses to analyze your repository
--   [Adjust the quality settings](../repositories-configure/adjusting-quality-settings.md) to control quality gate that Codacy applies to your commits and pull requests
-
-## You're all set! 🎉 {: id="all-set"}
+## What next?
```

Comment:
We call this "Next steps" on the other pages. Not sure what's best or if we even want to be consistent on this

Reply:
We'll take care of this final "polishing" at a later stage. :wink: